app/controllers/logs_controller.rb

```diff
@@ -393,38 +393,38 @@ def export
 
   private
 
-  def parse_and_create_log_parts(params, log)
-    ret = []
-    params['log_parts'].each{ |dc, lpdata|
-      lpdata['weight'] = nil if lpdata['weight'].strip == ''
-      lpdata['count'] = nil if lpdata['count'].strip == ''
-      next if lpdata['id'].nil? and lpdata['weight'].nil? and lpdata['count'].nil?
-      lp = lpdata['id'].nil? ? LogPart.new : LogPart.find(lpdata['id'].to_i)
-      lp.count = lpdata['count']
-      lp.description = lpdata['description']
-      lp.food_type_id = lpdata['food_type_id'].to_i
-      lp.log_id = log.id
-      lp.weight = lpdata['weight'].to_f
-      ret.push lp
-      lp.save
-    } unless params['log_parts'].nil?
-    ret
-  end
+    def parse_and_create_log_parts(params, log)
+      ret = []
+      params['log_parts'].each{ |_dc, lpdata|
+        lpdata['weight'] = nil if lpdata['weight'].strip == ''
+        lpdata['count'] = nil if lpdata['count'].strip == ''
+        next if lpdata['id'].nil? and lpdata['weight'].nil? and lpdata['count'].nil?
+        lp = lpdata['id'].nil? ? LogPart.new : LogPart.find(lpdata['id'].to_i)
+        lp.count = lpdata['count']
+        lp.description = lpdata['description']
+        lp.food_type_id = lpdata['food_type_id'].to_i
+        lp.log_id = log.id
+        lp.weight = lpdata['weight'].to_f
+        ret.push lp
+        lp.save
+      } unless params['log_parts'].nil?
+      ret
+    end
```

Comment:
I don't _think_ we want to indent these methods. I'm getting a rubocop error when running against this branch:


```
app/controllers/logs_controller.rb:396:5: C: Inconsistent indentation detected.
    def parse_and_create_log_parts(params, log) ...
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

May be a result of merging/rebasing?